src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,33 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
```

Comment:
Don't begin class or method implementation with empty line. Please remove the empty lines above the class declaration.